core/state-machine/src/testing.rs

```diff
@@ -43,44 +48,65 @@ impl<H: Hasher> TestExternalities<H> {
 	/// Create a new instance of `TestExternalities`
 	pub fn new_with_code(code: &[u8], mut inner: HashMap<Vec<u8>, Vec<u8>>) -> Self {
 		let mut overlay = OverlayedChanges::default();
+
 		super::set_changes_trie_config(
 			&mut overlay,
 			inner.get(&CHANGES_TRIE_CONFIG.to_vec()).cloned(),
 			false,
 		).expect("changes trie configuration is correct in test env; qed");
 
 		inner.insert(HEAP_PAGES.to_vec(), 8u64.encode());
+		inner.insert(CODE.to_vec(), code.to_vec());
 
-		TestExternalities {
-			inner,
+		let mut t = TestExternalities {
+			overlay,
 			changes_trie_storage: ChangesTrieInMemoryStorage::new(),
-			changes: overlay,
-			code: Some(code.to_vec()),
+			backend: InMemory::default(),
+			_hasher: Default::default(),
+		};
+
+		for (key, value) in inner {
```

Comment:
I just copy pasted from `core/state-machine/src/ext.rs`
But all in all now I think it might event be better to just change test externalities signature and make use of ext directly.. but that would imply adding a few lines in most tests

Reply:
done